projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json

```
@@ -0,0 +1,38 @@
+{
+  "block.roberts_collection.aki_hayakawa": "早川秋",
+  "block.roberts_collection.aloy": "埃洛伊",
+  "block.roberts_collection.captain_jack_sparrow": "杰克·斯帕罗船长",
+  "block.roberts_collection.chun_li": "春丽",
+  "block.roberts_collection.cloud_strife": "克劳德·斯特莱夫",
+  "block.roberts_collection.crash": "古惑狼",
+  "block.roberts_collection.denji": "电次",
+  "block.roberts_collection.dipper_pines": "迪普·派恩斯",
+  "block.roberts_collection.ellie_williams": "艾莉·威廉姆斯",
+  "block.roberts_collection.ezio_auditore": "艾吉奥·奥迪托雷",
+  "block.roberts_collection.himeno": "姬野",
+  "block.roberts_collection.isaac_clarke": "艾萨克·克拉克",
+  "block.roberts_collection.joel_miller": "乔尔·米勒",
+  "block.roberts_collection.john_wick": "约翰·威克",
+  "block.roberts_collection.kratos": "奎托斯",
+  "block.roberts_collection.lara_croft": "劳拉·克罗夫特",
+  "block.roberts_collection.leon_s_kennedy": "里昂·斯科特·肯尼迪",
+  "block.roberts_collection.link": "林克",
+  "block.roberts_collection.mabel_pines": "梅宝·派恩斯",
+  "block.roberts_collection.makima": "玛奇玛",
+  "block.roberts_collection.mario_bros": "马力欧",
+  "block.roberts_collection.master_chief": "士官长",
+  "block.roberts_collection.mega_man": "洛克人",
+  "block.roberts_collection.pacman": "吃豆人",
+  "block.roberts_collection.pong": "乓",
+  "block.roberts_collection.power": "帕瓦",
+  "block.roberts_collection.princeof_persia": "波斯王子",
+  "block.roberts_collection.ryu": "隆",
+  "block.roberts_collection.samus_aran": "萨姆斯·阿兰",
+  "block.roberts_collection.simon_ghost_riley": "西蒙 \"幽灵\" 莱利",
+  "block.roberts_collection.solairede_astora": "索拉尔",
```

```suggestion
  "block.roberts_collection.solairede_astora": "亚斯特拉的索拉尔",
```
原文为全称，故需要全文译出
<s>赞美太阳！</s>